app/components/UI/FiatOnRampAggregator/components/TransactionDetails.tsx

```diff
@@ -244,12 +277,12 @@ const TransactionDetails: React.FC<Props> = ({ order }: Props) => {
 					<ListItem.Amount>
 						<Text small bold primary>
 							{order.data.fiatCurrency.denomSymbol}
-							{order.amount} {order.data.fiatCurrency.symbol}
+							{renderFiat(order.amount, order.data.fiatCurrency.symbol, order.data.fiatCurrency.decimals)}
 						</Text>
 					</ListItem.Amount>
 				</ListItem.Content>
 				{order.state === SDK_ORDER_STATUS.Completed && (
-					<TouchableOpacity>
+					<TouchableOpacity onPress={() => handleLinkPress('https://etherscan.io/tx/' + order.data.txHash)}>
```

Comment:
You need for the blockchain explorer to factor for multi-chain support, have a look on: useBlockExplorer hook.

Reply:
Oh right, you have to be in that network to see that order. It's time to promote `useBlockExplorer` to `Base/hooks` and use it here :)